src/torrent.cpp

```diff
@@ -8453,7 +8447,8 @@ namespace libtorrent
 			m_seeding_time += m_ses.session_time() - m_became_seed;
 
 		if (is_finished())
-			m_finished_time += m_ses.session_time() - m_became_finished;
+			m_finished_time += duration_cast<seconds>(aux::time_now()
```

Comment:
you missed the linebreak, its the duration between now and became finished. And this is a duration<long int, std::ratio<1l, 1000000000l>

But the + operator wants a duration<long int>